src/pytest/test_followhashes.py

```diff
@@ -17,7 +19,7 @@ def testSyntax1(env):
             
     env.expect('ft.create', 'idx2',
             'LANGUAGE', 'eng' 
-            'SCHEMA', 'foo', 'text').equal('Unknown argument `foo`')
+            'SCHEMA', 'foo', 'text').equal('Invalid language')
```

Comment:
Why was this changed?

Reply:
before language was the name of the field so any input was valid.